<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxInClass</class>
 <widget class="QMainWindow" name="BoxInClass">
  <property name="windowTitle">
   <string>BoxIn</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="commandLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>340</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="commandLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>  Command:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="displayFeed">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="feedbackBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome to BoxIn v0.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>599</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BoxIn.qrc"/>
 </resources>
 <connections/>
</ui>
